sys/include/net/gnrc/ipv6/blacklist.h

```diff
@@ -30,12 +30,19 @@
 extern "C" {
 #endif
 
+/**
+ * @defgroup    net_gnrc_ipv6_blacklist_conf GNRC IPv6 address blacklisting compile configurations
```

Comment:
Can't really split the line, otherwise doxygen renders it as `@brief`. It is still within the absolute maximum though